File: java/dayOne/levelOne/TaskFour.java

import java.util.Arrays;
public class TaskFour {

    public static int [] getPerfectSquare(int [] numbers){
        
            int [] perfectSquare = new int [numbers.length];

        for(int count = 0; count < numbers.length; count++){

            int number = numbers[count];
            
            if(number >= 0){
              int root = (int)Math.sqrt(number);
              
                if(root * root == number){
                    perfectSquare[count] = number;

                }  
                else{
                    perfectSquare[count] = -1;

                }     
            
          } 
                    
      } 
                
      return perfectSquare;
    }

}

File: python/day_one/level_one/task_four.py
import math
def get_non_Perfect_square(numbers):

    Perfect_square_list = []   
    
    for number in numbers:
    
        if number >= 0:
        
            root = int(math.sqrt(number))
            
            if root * root == number:
            
                Perfect_square_list.append(number)
                
            else:
                Perfect_square_list.append(-1)     


    return Perfect_square_list 



